code/modules/jobs/job_types/clown.dm

```diff
@@ -0,0 +1,57 @@
+/datum/job/clown
+	title = "Clown"
+	flag = CLOWN
+	department_head = list("Head of Personnel")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the head of personnel"
+	selection_color = "#dddddd"
+
+	outfit = /datum/outfit/job/clown
+
+	access = list(ACCESS_THEATRE)
+	minimal_access = list(ACCESS_THEATRE)
+
+	display_order = JOB_DISPLAY_ORDER_CLOWN
+
+
+/datum/job/clown/after_spawn(mob/living/carbon/human/H, mob/M)
+	. = ..()
+	H.apply_pref_name("clown", M.client)
+
+/datum/outfit/job/clown
+	name = "Clown"
+	jobtype = /datum/job/clown
+
+	belt = /obj/item/pda/clown
+	ears = /obj/item/radio/headset/headset_srv
+	uniform = /obj/item/clothing/under/rank/clown
+	shoes = /obj/item/clothing/shoes/clown_shoes
+	mask = /obj/item/clothing/mask/gas/clown_hat
+	l_pocket = /obj/item/bikehorn
+	backpack_contents = list(
+		/obj/item/stamp/clown = 1,
+		/obj/item/reagent_containers/spray/waterflower = 1,
+		/obj/item/reagent_containers/food/snacks/grown/banana = 1,
+		/obj/item/instrument/bikehorn = 1,
+		)
+
+	implants = list(/obj/item/implant/sad_trombone)
+
+	backpack = /obj/item/storage/backpack/clown
+	satchel = /obj/item/storage/backpack/clown
+	duffelbag = /obj/item/storage/backpack/duffelbag/clown //strangely has a duffel
+
+	box = /obj/item/storage/box/hug/survival
+
+	chameleon_extras = /obj/item/stamp/clown
+
+/datum/outfit/job/clown/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+
+	H.fully_replace_character_name(H.real_name, pick(GLOB.clown_names)) //rename the mob AFTER they're equipped so their ID gets updated properly.
+	H.dna.add_mutation(CLOWNMUT)
```

Comment:
as much as i hate the smile mutation you are removing it when we have it please put it back or give a good reason for removal